10_code/SF_LA_analysis.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd 
+import datetime
+
+###### read in data ###### 
+###### ###### ###### #####
+
+daily_cases = pd.read_csv("../00_source_data/us-counties.csv")
+daily_cases.head()
+
+
+##### subset county and 3 week span #####
+#########################################
+
+### SF ###
+##########
+# policy date for shelter in place
+sf_policy = datetime.datetime.strptime('16032020', "%d%m%Y").date()
+date_before = [sf_policy - datetime.timedelta(days=x) for x in range(numdays+1)]
+date_after = [sf_policy + datetime.timedelta(days=x) for x in range(numdays+1)]
+sf_date_list = date_before + date_after
```

Comment:
I noticed you are comparing LA county and SF county. Could you provide more comments on why choosing these two counties?(Are they similar in terms of population, geographic location, and climate?) Also could you please comment on why you decided to use the 6-week span?

Reply:
My reasoning was:
- They have similar population
- Both relatively wealthy and have access to medial resources
- Both are international metropolitan, so likely to be hit by Covid quickly.
- 6 weeks seemed not too long as to include the spike of cases after increased testing accessibility; also not too short to mask the clinical latency of the virus. But it's a rough guess, and open to suggestion.

Reply:
6 weeks does make sense if we want to take the latency into the consideration. But I believe it could be better to make the post-announcement  a bit longer (~ 4 weeks) for the following reason:

i.e if a county decides to roll out a remote-work order, there are potential problems: 
 - different companies could implement it at a different rate (though most companies should act immediately)
 - assume a company asked its employees to work from home the day after the state-wide policy announcement, taking the 2-week latency period into consideration, I'm not sure the remaining 1 week is enough to observe significant changes in COVID cases. 

However, it is also important to make sure there is no overlapping of policies during the period of analysis, which could shadow the true causal effect of the target policy.